module/metrics/compliance.go

```diff
@@ -49,6 +53,33 @@ func NewComplianceCollector() *ComplianceCollector {
 			Help:      "the final view of the committed epoch with the greatest counter",
 		}),
 
+		currentEpochFinalView: promauto.NewGauge(prometheus.GaugeOpts{
+			Name:      "current_epoch_final_view",
+			Namespace: namespaceConsensus,
+			Subsystem: subsystemCompliance,
+			Help:      "the final view of the current epoch with the greatest counter",
```

Comment:
```suggestion
			Help:      "the final view of the current epoch",
```

I think the "with the greatest counter" part is a bit confusing, since it is referring to something else in the other metric where that language is used. There is only ever one current epoch so leaving it out should be OK.